### PR TITLE
Update docs about ports, fix nits

### PR DESCRIPTION
**Pull request by thockin**
Hopefully this is a clarification.

@bgrant0607

### CODE REVIEW

**Comment by thockin** [comment]
@k8s-bot unit test this github issue: #IGNORE

**Comment by lavalamp** [comment]
Actually, since you tagged @bgrant0607 I guess I'll let him look :)

File: pkg/api/v1/types.go [hunk 1]
```diff
@@ -1022,11 +1022,16 @@ type Container struct {
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/containers.md#containers-and-commands
 	Args []string `json:"args,omitempty"`
 	// Container's working directory.
-	// Defaults to Docker's default. D
-	// efaults to image's default.
+	// Defaults to Docker's default, which may be specified by the container
```
Comment (bgrant0607) [inline pkg/api/v1/types.go]: I'm trying to reserve the term "Default" for the way we explicitly initialize fields during the defaulting pass.

If the field will remain empty if unspecified, I've recently suggested "inferred to be...". In this case, that doesn't work perfectly.

How about: "If unspecified, uses container runtime's default, which may be specified by the container"

File: pkg/api/v1/types.go [hunk 2]
```diff
@@ -1022,11 +1022,16 @@ type Container struct {
...
+	// image.
 	// Cannot be updated.
 	WorkingDir string `json:"workingDir,omitempty"`
-	// List of ports to expose from the container.
+	// List of ports to expose from the container. Exposing a port here gives
+	// the system additional information about the network connections a
+	// container uses, but is primarily informational. Not specifying a port here
+	// DOES NOT prevent that port from being exposed. Any port which is
+	// listening on the default "0.0.0.0" address inside a container will be
+	// accessible from the network, by default.
```
Comment (bgrant0607) [inline pkg/api/v1/types.go]: Does the "by default" text at the end add anything? Is there any way to override that behavior? If not, I'd drop those 2 words.

**Comment by thockin** [comment]
Word nits fixed and repushed

**Comment by thockin** [comment]
I'm not rebasing this again until it is LGTM.

**Comment by thockin** [comment]
Looking for LGTM

**Comment by thockin** [comment]
I have rebased again.  I'm out of town this week, so I am going to manually merge this, and any niggles can be fixed ex post facto.

**Pull request merged**